app/code/Magento/MediaGallery/Test/Unit/Model/Keyword/Command/SaveAssetLinksTest.php

```diff
@@ -56,6 +63,8 @@ public function setUp(): void
      * @param int $assetId
      * @param array $keywordIds
      * @param array $values
+     *
+     * @throws CouldNotSaveException
```

Comment:
I guess the `@throws` tag is redundant for test methods, considering the exception is actually never thrown.